ras_party/models/models.py

```diff
@@ -390,6 +390,14 @@ class Enrolment(Base):
         ),
     )
 
+    def to_dict(self) -> dict:
+        return {
+            "business_id": self.business_id,
+            "respondent_id": self.respondent_id,
```

Comment:
Very debatable if status of respondent_id should be passed back

Reply:
As discussed off-line, I don't think the API should return the internal PK of the database record (i.e. in this case the PK of the respondent record).

However, I acknowledge that if this is omitted multiple respondent enrollments for the same business/survey will appear as duplications with no differentiation between each. It's only the inclusion of the `respondent_id` that's providing uniqueness as an enrolment is scoped by respondents. The ideal solution for this API would be to return the respondent party UUID instead of the db PK (even if it's not needed for the current use cases being addressed).

At the very least we should document/capture that we are intending a future extension of the `/enrolments` API to cover wider use cases to that may require the UUID of the respondent to be included.

This new approach is not a direct replacement for the list of `"associations": [...]` in the current API, it is a step towards breaking up the bloated requests/responses that are causing performance issues in high frequency pages.